crates/kitsune_p2p/crates/bin_data/src/lib.rs

```diff
@@ -197,10 +197,18 @@ impl From<Vec<u8>> for KitsuneOpData {
 
 impl std::fmt::Debug for KitsuneOpData {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        f.write_fmt(format_args!(
-            "KitsuneOpData({})",
-            &holochain_util::hex::many_bytes_string(self.0.as_slice())
-        ))
+        let data = if self.0.len() > 32 {
```

Comment:
Please add a comment that specifies what this outputs.

Reply:
Yep, good idea. Added a comment